atom/src/atomlist.cpp

```diff
@@ -739,7 +779,24 @@ class AtomCListHandler : public AtomListHandler
 #else
         static char *kwlist[] = { "key", "reverse", 0 };
 #endif
+#if PY_VERSION_HEX >= 0x03070000
+        int nargs = (int)PyTuple_GET_SIZE( args );
+        PyObject **stack = &PyTuple_GET_ITEM( args, 0 );
+
+        PyObject *const *stackbis;
+        PyObject *kwnames;
+        if (_PyStack_UnpackDict(stack, nargs, kwargs, &stackbis, &kwnames) < 0) {
+            return NULL;
```

Comment:
Should this return `0`?

Reply:
Both are fine and I simply more or less copy pasted part of CPython C code but for homogeneity in atom 0 may be more appropriate.